Fix: Fix to show color when user join the chat

MilaCodesIt opened the pull request:
Screenshot:
<img width="271" height="86" alt="image" src="https://github.com/user-attachments/assets/7452c32e-d681-4e41-9e8d-af8029a2808c" />

Front-end change, as it is related to styling, translations were not affected.
This change has been tested in localhost, if any issues please let me know.

## Double check

- [x] I included a screenshot, logs, or example payload to demonstrate the change.
- [x] This is a frontend change and it supports [translations](https://docs.owncast.dev/web-translations), or it's not a frontend change.
- [x] This change has actually been tested.
- [x] The code has been run through the proper linters and/or formatters for the language.
- [x] There is an issue discussing this change and it's assigned to me.
- [x] I read the "Read first" details about filing this PR.

---

# Read first

## We manage everything through issues, not PRs.

A PR is a **solution**. A solution without a **problem** breaks release notes. **Please open an issue** so the list of problems that were fixed can be correctly listed and be attributed to you.

If this is an unsolicited change, or there is no existing issue filed for it, **please open a GitHub issue before creating a pull request**. This will allow us to discuss the motivations and the big picture behind the change first. It's possible there may be other solutions that should be discussed for what you think should be built. It is possible your change will be rejected unless some discussion around your proposal happens first. While creating this PR means you probably already did the work, **it still makes sense to file an issue now**, and into the future when you have proposed changes.

Additionally, when attributing credit in releases, we use issues to determine who worked on each version. If there isn't an issue assigned to you, you may not get credit in release notes.

## Resources

- [Developing Owncast](https://docs.owncast.dev/development)
- [How We Develop Frontend React Components](https://docs.owncast.dev/develop-frontend-components)
- [Supporting Translations](https://docs.owncast.dev/web-translations)

## Description

Once there is an issue filed, a PR can be linked to it.

Please include a summary of the change in the PR and which issue number is fixed, including relevant motivation and context. Mark this as a Draft or WIP and write up some details later and start a conversation, even if your PR is not ready for review or you haven't yet provided all the information.

Fixes # (issue)

## Screenshot Examples or Logs

If this is a frontend change, please include a screenshot of the change. If this is a backend change, please include relevant logs or examples of the change in action if applicable.

---

Some things you might want to mention:

1. Why are you making the change?
2. Explain how it works and decisions you made.
3. If you're fixing something, what was wrong? How should we stop from having this issue happen again?
4. If this is a new feature or addition to functionality, why should it be added? What are the use cases? Who was asking for this functionality?
5. If this is a frontend change, does the text support [translation](https://owncast.notion.site/web-translations)?
   
Thank you so much for contributing to Owncast! 🎉

MilaCodesIt committed:
Fix: Fix to show color when user join the chat

MilaCodesIt committed:
Changes to restore unnecesary changes

MilaCodesIt committed:
Deleting unnecesary changes

MilaCodesIt committed:
Identation issues fixed

gabek commented:
If you could, update your branch with `develop` so the tests will pass. You can press "Update branch" and then pull down those changes, or rebase, or any way you like.

MilaCodesIt commented:
As I forked the original repo, I just updated and rebased the current branch so it takes latest changes.

It just needs to finish the checks and it can be merged.

MilaCodesIt commented:
ready for merge.

gabek committed:
fix(js): run prettier against changed code

gabek commented:
Thank you so much!
